Test test_max_min_bool_cpu fails due to a known issue:
> https://github.com/pytorch/pytorch/pull/185970

Author: Alexander Grund (TU Dresden)
diff --git a/test/inductor/test_torchinductor.py b/test/inductor/test_torchinductor.py
--- a/test/inductor/test_torchinductor.py
+++ b/test/inductor/test_torchinductor.py
@@ -12632,6 +12632,7 @@ def forward(self, arg0_1: "Sym(s77)", arg1_1: "Sym(s27)", arg2_1: "Sym(s53)", ar
         self.common(fn, (t1,))
 
     @requires_cuda
+    @skip_if_cpu
     def test_max_min_bool(self):
         # Regression test for https://github.com/pytorch/pytorch/issues/174069
         # max/min on boolean tensors returned incorrect indices in Triton
